### ISSUE TITLE
Doesnt work properly

### ISSUE THREAD

**Issue by wodsuz** [orsifrancesco/instagram-without-api-node#6]
Heya, this doesnt work now right? 

It can only fetch old  posts on some instagram accounts - https://www.instagram.com/albertoangela/ and this is also broken https://orsi.me/sniffagram/

**Comment by orsifrancesco** [comment]
@wodsuz you are right, after long time I fixed https://orsi.me/sniffagram/ (..and probably it is a temporary solution) but instagram moved the pictures array to another endpoint (with more than one redirect) making harder to catch images without scrapers like Puppeter ( https://pptr.dev/ ).
If I have time maybe one day I will create a new version, but judging by the many demands without any kind of gratitude from the community, I have lost the desire.
